### DISCUSSION TITLE
Slider_Circle

### DISCUSSION THREAD

**Discussion by CODE-Yasser** [ocornut/imgui#5350] [New Users Build/Link/Run issues ONLY!]
Hi,
how can i make slider circle like this:

![Screenshot_4](https://user-images.githubusercontent.com/67662288/170496029-a2e022e3-9c24-47ac-b45e-40448e5cebdd.png)
![unknown](https://user-images.githubusercontent.com/67662288/170496048-fe70c4dc-2506-4e4c-a906-933905fae5c8.png)

sorry, i'm new here

**Comment by ocornut** [comment]
Sorry but #1586
